sahi/predict.py

```diff
@@ -165,6 +166,9 @@ def get_sliced_prediction(
             0: no print
             1: print number of slices (default)
             2: print number of slices and slice/prediction durations
+        merge_buffer_length: int
+            The length of buffer for slices to be used during sliced prediction, which is suitable for low memory.
+            It may affect the AP if it is specified.
```

Comment:
```suggestion
            It may affect the AP if it is specified. The higher the amount, the closer results to the non-buffered
            scenario. See [the discussion](https://github.com/obss/sahi/pull/445).
```

Reply:
@weypro @fcakyon can you check the docstring and commit if it's okay ?